roles/matrix-bot-maubot/templates/systemd/matrix-maubot.service.j2

```diff
@@ -0,0 +1,40 @@
+#jinja2: lstrip_blocks: "True"
```

Comment:
Renaming this file to `matrix-bot-maubot.service` would be preferable.

Not just the template, but the file on the Matrix server as well.

Don't forget to update `init.yml` too, so that it points to `matrix-bot-maubot.service`.

Reply:
Done